backend/src/schema/resolvers/todos.js

```diff
@@ -0,0 +1,50 @@
+const { decodedToken } = require("../../utils/decode");
+
+const { todoListData } = require("../../data");
+
+module.exports.TodoResolver = {
+  Todo: {
+    author(todo) {
+      return todoListData.find(value => {
+        return todo.id === value.id;
+      }).author;
+    }
+  },
+  Query: {
+    todos: (object, params, context) => {
+      const decoded = decodedToken(context.token);
+      return todoListData.filter(todo => todo.author.name === decoded.username);
+    }
+  },
+  Mutation: {
+    delToDo: (object, params, context) => {
```

Comment:
```suggestion
    deleteTodo: (object, params, context) => {
```
I prefer not to use abbreviations.